hack/release.sh

```diff
@@ -297,7 +303,7 @@ release_index() {
 }
 
 main() {
-	build_all
+	[ "$SKIP_RELEASE_BUILD" -eq 1 ] || build_all
```

Comment:
@andrewhsu out of curiosity where is this env set? is it just a knob to be able to push without rebuilding?